### ISSUE TITLE
Pseudo-Augustinus is recognized as Ambrosiaster

### ISSUE THREAD

**Issue by PonteIneptique** [OpenGreekAndLatin/csel-dev#305]
It seems well accepted : 

Source : http://www.persee.fr/doc/thlou_0080-2654_2010_num_41_2_3829_t1_0275_0000_3
- urn:cts:latinLit:stoa0040a.stoa002.opp-lat1
- urn:cts:latinLit:stoa0040a.stoa003.opp-lat1

**Comment by AlisonBabeu** [comment]
You are quite right @PonteIneptique, I will create an authority record for Ambrosiaster and fix this data within the Perseus Catalog and the cite_collection  tables by redirecting these URNs.

**Comment by PonteIneptique** [comment]
Yay !

**Comment by AlisonBabeu** [comment]
So while the catalog data was updated for this it has never been updated for this in Scaife CSEL files, something I can now do so I'm going to finally do it!
